docs/CONTRIBUTING.md

```diff
@@ -61,25 +61,6 @@ yarn prepare
 
 5.  **Setup environmental variables:** To run E2E Tests, you should set the following variables in your `.env` file in the repository’s root.
 
-```bash
```

Comment:
is this file the same as the one in the root directory? if so, should we remove this one completely so as to not duplicate?

Reply:
now I see that this may be needed to build the docs site, if so disregard above

Reply:
If you look at `package.json`, the `docs:build` script copies that and it is done automatically.
Therefore this note does not relate to this PR and should be solved with a new documentation (external UI).